code/src/components/Form.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+import Header from "components/Header";
+import Checkbox from "components/Checkbox";
+import Text from "components/Text";
+import DropDown from "components/DropDown";
+import RadioButton from "components/RadioButton";
+import Submit from "components/Submit";
+import Summary from "components/Summary";
+import Reload from "components/Reload";
+
+import "./form.css";
+
+const Form = () => {
+  //all state hooks
+  const [displayForm, setDisplayForm] = useState(true);
+  const [displaySummary, setDisplaySummary] = useState(false);
+  const [checkBox, setCheckBox] = useState([]);
+  const [text, setText] = useState("");
+  const [distance, setDistance] = useState({});
+  const [answer, setAnswer] = useState("");
+
+  //all state functions
+  const handleCheckBoxChange = (newCheck) => {
+    checkBox.includes(newCheck)
+      ? setCheckBox(checkBox.filter(item => item !== newCheck))
+      : setCheckBox([...checkBox, newCheck]);
+  };
+
+  const handleTextChange = (newText) => {
+    setText(newText);
+  };
+
+  const handleDistanceChange = (newDistance) => {
+    setDistance(newDistance);
+  };
+
+  const handleAnswerChange = (newAnswer) => {
+    setAnswer(newAnswer);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setDisplayForm(false);
+    setDisplaySummary(true);
+  };
+
+  return (
+    <form className="app-container" onSubmit={handleSubmit}>
+      {displayForm && (
+        <div className="form-container">
```

Comment:
Interesting to have the form tag outside of the conditional and wrapping both questions and the summary, with a div tag that sets the design of the form. Does the summary need to be inside the form tag? Otherwise maybe wrap the whole thing in a section and just the form itself in a form tag, just to make it a little bit easier to follow along. 😊